elements/glossy-portfolio-theme/lib/glossy-portfolio-home.js

```diff
@@ -30,7 +30,7 @@ export class GlossyPortfolioHome extends DDDSuper(I18NMixin(LitElement)) {
       this.__disposer.push(reaction);
     });
     autorun((reaction) => {
-      this.backgroundImage = toJS(store.themeData.variables.image) || "https://preview.redd.it/ppbbhgt5xof71.jpg?width=1080&crop=smart&auto=webp&s=4e51c19ae4ece47b329052b99b48494bd86954aa";
+      this.backgroundImage = toJS(store.themeData.variables.image);
```

Comment:
I'd verify that this is there first to avoid a possible JS error if the variable isn't defined or setting the image to nothing if the site has none defined